Running ooffice gives "Try `uname --help' for more information" before starting
Fix provided by Eric Anderson 
http://bugs.debian.org/196039

Index: oo_cvs/setup2/script/sopatchlevel.sh
===================================================================
RCS file: /cvs/installation/setup2/script/sopatchlevel.sh,v
retrieving revision 1.4
diff -u -3 -p -r1.4 sopatchlevel.sh
--- oo_cvs/setup2/script/sopatchlevel.sh	5 Nov 2001 10:30:59 -0000	1.4
+++ oo_cvs/setup2/script/sopatchlevel.sh	4 Jun 2003 09:34:23 -0000
@@ -64,7 +64,7 @@
 sd_inst="`dirname "$0"`"
 sd_platform=`uname -s`
 
-if [ "$sd_platform"="SunOS" ]; then
+if [ "$sd_platform" = "SunOS" ]; then
 	#
 	# check required patchid
 	#
